suitcase/tests/test_hdf5.py

```diff
@@ -56,6 +61,56 @@ def test_hdf5_export_single():
     shallow_header_verify(fname.name, hdr)
 
 
+@pytest.mark.xfail(reason='name is not included as a key at descriptor'
```

Comment:
Please write tests that will pass.  Update the synthetic data generation if needed.